programs/token-wrapper/src/lib.rs

```diff
@@ -135,6 +314,7 @@ pub mod token_wrapper {
 enum CalleeProgram {
     TokenStar,
```

Comment:
nit: Could you rename this to `SplToken`? I was a bit confused, but that might just be because we called `Token*` the interface version for a time